crates/api_crud/src/community/create.rs

```diff
@@ -73,6 +74,10 @@ pub async fn create_community(
     is_valid_body_field(desc, false)?;
   }
 
+  if data.visibility == Some(lemmy_db_schema::CommunityVisibility::Private) {
+    is_admin(&local_user_view)?;
```

Comment:
Only admins can create private communities? Makes sense I guess, as long as that's documented somewhere.

Reply:
Otherwise it would be possible to create a private community and invite only friends (or bots) in order to post content that violates site rules. Admins would only notice this if they come across the content by chance on the frontpage which is unlikely on a big instance. In the future we can make an approval process for community creation, to ensure that private communities comply with site rules.

Reply:
Makes sense.